iocore/net/P_QUICNetVConnection.h

```diff
@@ -171,6 +171,9 @@ class QUICNetVConnection : public UnixNetVConnection, public QUICConnection, pub
   // QUICNetVConnection
   void registerNextProtocolSet(SSLNextProtocolSet *s);
 
+  int select_next_protocol(SSL *ssl, const unsigned char **out, unsigned char *outlen, const unsigned char *in,
```

Comment:
I want to put this line here, but for now, please move it under `// QUICConnection` since it's defined in QUICConnectionInfoProvider.